lib/bundle.js

```diff
@@ -481,6 +482,33 @@ function findReferenceByMainKeyInTraceFromContext(documentContext, mainKeyInTrac
   return relatedRef;
 }
 
+/**
+ * Verifies if a node has same content than one of the parents so it is a circular ref
```

Comment:
`Verifies if a node has same content as one of the parents so it is a circular ref`

than --> as